app/templates/components/forms/wizard/sessions-speakers-step.hbs

```diff
@@ -1,6 +1,6 @@
 <form class="ui form" autocomplete="off" {{action 'moveForward' on='submit' preventDefault=true}}>
 
-  <div class="ui three column centered grid">
+  <div class="ui centered grid">
```

Comment:
I have removed the column numbers so that the content can take as much space as it requires by centrally aligning itself on any sized screen.